<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1114</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="AutoBtn">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>490</y>
      <width>211</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>27</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>自动追踪</string>
    </property>
   </widget>
   <widget class="QLabel" name="DispLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::WinPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>0</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>匹配位置：</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>19</pointsize>
      </font>
     </property>
     <property name="text">
      <string>x:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>60</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>y:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="x_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="y_label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>160</y>
      <width>191</width>
      <height>161</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="title">
     <string>搜索偏移量：</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>x_offset:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>y_offset:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="x_offset">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="y_offset">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>17</pointsize>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dir:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="DirLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>330</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="AgentAction">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>281</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>400</y>
      <width>471</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>模拟验光仪</string>
    </property>
    <widget class="QPushButton" name="OpenFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>打开图片</string>
     </property>
    </widget>
    <widget class="QPushButton" name="UpBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>90</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="LeftBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>160</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>left</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RightBtn">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>160</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Right</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DownBtn">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>210</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Down</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="StepLineEdit">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>160</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Step:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1114</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
